scripts/dummy.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 
 const lr = require('load-runner')();
-const Promise = require('bluebird');
```

Comment:
While it's true that this is not actually needed, we put it there because bluebird is more efficient than the built-in Promises implementation.